zzz_generated/darwin-framework-tool/zap-generated/cluster/Commands.h

```diff
@@ -4143,7 +4145,7 @@ class WriteLevelControlOptions : public WriteAttribute {
         __auto_type * params = [[MTRWriteParams alloc] init];
         params.timedWriteTimeout = mTimedInteractionTimeoutMs.HasValue() ? [NSNumber numberWithUnsignedShort:mTimedInteractionTimeoutMs.Value()] : nil;
         params.dataVersion = mDataVersion.HasValue() ? [NSNumber numberWithUnsignedInt:mDataVersion.Value()] : nil;
-        NSNumber * _Nonnull value = [NSNumber numberWithUnsignedChar:mValue];
+        NSNumber * _Nonnull value = [NSNumber numberWithUnsignedChar:mValue.Raw()];
```

Comment:
This is an interesting diversion... this attribute is of type `OptionsBitmap`, which is triggering the template conditional `{{#if_is_strongly_typed_bitmap type}}`, which accesses the value using `.Raw()`

The original code does not have this distinction - was that an oversight? Or does this end up being a no-op in practice @bzbarsky-apple?

Reply:
So it turns out this difference is actually necessary, it's unclear why when processing `mValue` we treat types differently than when processing deeply-nested types

Aborting this PR

Reply:
I don't know why d-f-t is using the types it's using offhand, but what needs to happen is that the type declarations need to match the conversions.....  Presumably that's the case now, and changing one without the other is obviously not going to work.

Reply:
@gmarcosb See above.